upload/enqueue_test.go

```diff
@@ -44,7 +44,7 @@ func TestAlbumId(t *testing.T) {
 	for _, tt := range testData {
 		t.Run(tt.name, func(t *testing.T) {
 			job.AlbumName = tt.in
-			got := job.albumID()
+			got, _ := job.albumID()
```

Comment:
Given that you have modified the contract of `job.albumID()`, it would be worth to add a new test case. In case that `job.albumID()` should return an `err != nil`.

What do you think?

Reply:
I've expanded the existing test to also check the returned error, let me know what you think!